images/x86_64/Ubuntu_14.04/shippable.yml

```diff
@@ -1363,14 +1363,92 @@ jobs:
                 - ARCHITECTURE: "x86_64"
                 - OS: "Ubuntu_14.04"
                 - DOCKER_VER: "1.13"
-                - MODIFY_KERNEL: "false"
+                - MODIFY_KERNEL: "true"
+                - KERNEL_VER: "3.13.0-153"
+          script:
+            - set -o pipefail
+            - |
+               export IMG_VER=$REL_VER
+               export REL_VER_DASH=${REL_VER//./-}
+               DOC_VER_DASH=${DOCKER_VER//./-}
+               KERNEL_VER_DASH=${KERNEL_VER//./-}
+               export FAM_NAME=$REL_VER_DASH"-"$DOC_VER_DASH"-"$KERNEL_VER_DASH"-patch-u14-x86-64"
```

Comment:
Family has changed to also include the kernel version. Will clean up the older images.